mars/oscar/backends/ray/pool.py

```diff
@@ -72,8 +72,9 @@ async def start_sub_pool(
         if not pg:
             bundle_index = -1
         # Hold actor_handle to avoid actor being freed.
+        num_cpus = config['kwargs'].get('sub_pool_cpus', 1)
         actor_handle = ray.remote(RaySubPool).options(
-            num_cpus=1, name=external_address, placement_group=pg,
+            num_cpus=num_cpus, name=external_address, placement_group=pg,
```

Comment:
Why does a sub pool use more than one cpu?

Reply:
The sub pool may use 1 cpu or 0 cpu, no more than 1 cpu.